scripts/log_reports.py

```diff
@@ -20,144 +20,135 @@
 
 from tabulate import tabulate
 
-
-MAX_LEN_MESSAGE = 2900  # slack endpoint has a limit of 3001 characters
+MAX_LEN_MESSAGE = 2900  # Slack endpoint limit of 3001 characters
 
 parser = argparse.ArgumentParser()
 parser.add_argument("--slack_channel_name", default="trl-push-ci")
 
-# Set up logging
-logging.basicConfig(level=logging.INFO)
-
-
-def main(slack_channel_name=None):
-    failed = []
-    passed = []
-
+# Set up logging using context manager
+def setup_logging():
+    logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+
+setup_logging()
```

Comment:
Why do you need it in a function?

Reply:
`log_report` is a script, so its code is not intended to be imported or reused. And there's no maintenance involved, it's just a line called once. Having a comment will do just fine.

```suggestion
# Setup logging
logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```